worker/util.go

```diff
@@ -100,3 +113,10 @@ func detectResources(conf config.Resources) config.Resources {
 // that doesn't do anything.
 func NoopTaskRunner(l TaskControl, c config.Worker, j *pbf.TaskWrapper) {
 }
+
+const (
+	b  = 1
+	kb = 1024 * b
+	mb = 1024 * kb
+	gb = 1024 * mb
```

Comment:
I should think before I write. Since gopsutil is returning bytes, I'm pretty sure the correct number to use is 1000.